SparkPost/2web2ui
# TR-2133 - Resolve issue with suppression list deletion when trying to delete multiple lists simultaneously

[TR-2133](https://jira.int.messagesystems.com/browse/TR-2133)

### What Changed
- Adds Cypress tests for Suppression List page
- Moves RV Cypress tests to a subfolder (`cypress/tests/integration/recipients`)
- Adds a few missing screen reader only labels to form elements within the suppression list flows (**NOTE:** Prettier reformatted these files as well - lots of noise, unfortunately)
  - `<DatePicker/>` component
  - `<FilterDropdown/>` component
  - `<SuppressionSearch/>` component
  - `<EmailSearch/>` component
- Refactors Suppression List `<Results/>` component from a Class to a functional component _(I created the tests with the existing component, ensured they were passing, refactored the component, and then ran the tests again)_
- Disables delete buttons within the suppression list table while a deletion is pending

### How To Test
- Try to reproduce the bug as outlined here under 'Steps to Reproduce' and be DEFEATED! 😈
https://jira.int.messagesystems.com/browse/TR-2133
- Ensure the Suppressions list page still functions in the same manner as production

### To Do
